src/network/dns.c

```diff
@@ -173,8 +173,13 @@ do_real_lookup(char *name, struct sockaddr_storage **addrs, int *addrno,
 	{
 		struct in_addr inp;
 
+#ifndef WIN32
 		if (is_ip_address(name, strlen(name)) && inet_aton(name, &inp))
 			hostent = gethostbyaddr(&inp, sizeof(inp), AF_INET);
+#else
+		if (is_ip_address(name, strlen(name)) && inet_pton(name, &inp))
```

Comment:
inet_pton has different declaration (arguments). Please add configure.ac check (if not yet such exists).
and in code add:
#ifdef HAVE_INET_PTON
 ... inet_pton
#else
... inet_aton
#endif
inet_pton is preferred.